app/position/positionkit.h

```diff
@@ -60,6 +60,9 @@ class PositionKit : public QObject
     // GPS satellites advanced data (Horizontal dilution of precision)
     Q_PROPERTY( double hdop READ hdop NOTIFY hdopChanged )
 
+    // GPS fix status
+    Q_PROPERTY( QString fix READ fix NOTIFY fixChanged )
```

Comment:
is "fix" common name for this? for me it sounds more like you want to fix your GPS connection or so?

Reply:
"fix" or "position fix" is a common term - see e.g. https://en.wikipedia.org/wiki/Geopositioning#Background or https://en.wikipedia.org/wiki/Time_to_first_fix

Reply:
I think so, @wonder-sk ?